client/src/App.tsx

```diff
@@ -1,7 +1,55 @@
 import React from "react";
 import "./App.scss";
 
+let socket = new WebSocket("ws://127.0.0.1:3010/ws");
```

Comment:
I think that all of this code might be able to be moved into inside the actual react App started in current line 50: `const App: React.FC = () => { ...`
You can have logic and variables in there before the return statement. @rgreen312 Can you make sure that sounds right, or is leaving this websocket code out here ok?

Reply:
@rgreen312 the gapi code functions are outside the react App, but I am not sure about react conventions for this

Reply:
you guys should just put whatever testing code you need in there now to get it working. The front end team and I will restructure the react code later